src/components/developer-preview.js

```diff
@@ -0,0 +1,253 @@
+            <li><strong>Measurement sets</strong>: A measurement set object represents a set of performance data related to one specific category (Promoting Interoperability, Improvement Activities, Quality), and is tied to a submission object.</li>
+            <li><strong>Measurements</strong>: A measurement object represents one single data point related to a specific measure in a given category, and is tied to a measurement set object.</li>
+          </ul>
+        </p>
+
+        <h2 className='ds-h2'>Authentication</h2>
+        <p className='ds-text'>Both the Developer Preview and the Production environments require a JSON Web Token ("API key") for authentication purposes.</p>
+
+        <a className='ds-c-button ds-c-button--primary' href='/qpp-submissions-docs/developer-preview#api-key'>Get an API Key</a>
+
+        <h2 className='ds-h2'>Notes</h2>
+        <ul>
+          <li>Any data submitted in the Developer Preview will not count toward a MIPS-eligible clinician or group's final score in the Quality Payment Program.</li>
+          <li>Since the Developer Preview is an integration environment for the Submissions API, it may be changed in ways that break backward compatibility and is not subject to any service-level guarantees or deprecation policy.</li>
+        </ul>
+
+        <h2 className='ds-h2'>Where to go for support</h2>
+        <ul>
+          <li>Send test API requests in the <a href='https://qpp-submissions-sandbox.navapbc.com/'>Interactive Documentation</a> using your Developer Preview API key.</li>
+          <li>Contact the QPP Service Center.</li>
+          <li>Post product feedback about the Developer Preview in the <a href='https://groups.google.com/forum/#!forum/qpp-apis'>Google Group for QPP APIs</a>.</li>
+        </ul>
+
+        <h1 className='ds-h1' id='api-key'>Get an API Key</h1>
+        <p className='ds-text'>This content is not yet available. Please check back later.</p>
+
+        <h1 className='ds-h1' id='authentication'>Authentication</h1>
+        <p className='ds-text'>You must authenticate your account when using the <a href='https://cmsgov.github.io/qpp-submissions-docs/developer-preview'>Developer Preview</a> (base url: https://preview.qpp.cms.gov/api/submissions/). Authenticate via bearer auth by adding your API token to the header of every request using the key value: <strong>Authorization: Bearer [YOUR API TOKEN]</strong>.</p>
+        <p className='ds-text'>API keys carry many privileges, and must not be shared in publicly accessible areas such as GitHub and in client-side code.  Even within organizations, access must be limited to staff embedding it in software.</p>
+        <p className='ds-text'>Your API key carries many privileges, so be sure to keep it secret! Do not share your secret API key in publicly accessible areas such GitHub, client-side code, and so forth.</p>
```

Comment:
these two paragraphs are repetitive, we could just keep one

Reply:
yup, good call out @kencheeto . @marimiyachi  - delete the 2nd paragraph.